virtio-bindings/import-linux-headers.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# SPDX-License-Identifier: (BSD-3-Clause OR Apache-2.0)
+# Copyright Red Hat, Inc.
+#
+# ./import-linux-headers.sh path/to/kernel/headers
+#
+# Import header files from a Linux kernel header tree. Be sure to run `cargo
+# build` to test that bindgen succeeds before committing the updated headers.
+
+src=$1
+dst=include/linux
+
+import_header() {
+	# Use our local header files rather than system headers
+	sed -e 's%#include <\([^>]*\)>%#include "\1"%' "$1" >"$2"
+}
+
+mkdir -p "$dst"
+
+# If you change this list, remember to update build.rs
+for header in \
+	virtio_blk \
```

Comment:
What about importing also version.h?
So we can easily figure out which linux version these headers belong to. (I also suggest writing it in the commit, I guess they are v6.12 as before, but I couldn't find references).

I mean just this change:

```suggestion
	version \
	virtio_blk \
```

Reply:
<linux/version.h> does not have a license header indicating it can be distributed under the BSD license.

Instead I updated the commit description to mention explicitly that the Linux 6.12 headers were used. Past updates to kernel headers were accompanied with a commit message that includes the kernel version. Let's keep doing that.

Reply:
@stefanha I see, makes sense to me.

BTW not sure if it was done by mistake, but in your last version I see `virtio-bindings/include/linux/version.h`

Reply:
Yes, it was a mistake. I had already implemented your idea when I noticed the licensing situation is not clear and forgot to remove version.h. I have pushed a fixed version now.